Sources/Utils.swift

```diff
@@ -32,9 +32,9 @@ public var logLevel: LogLevel = .Errors
 
 public var logger: (LogLevel, Any) -> Void = { print($1) }
 
-func log(level logLevel: LogLevel, _ message: Any) {
+func log(level logLevel: LogLevel, _ message: @autoclosure () -> Any) {
```

Comment:
how did you find out about this? just looking at the code? or did you use profiler? if profiler would be great to see how you profiled this

Reply:
I did it via the profiler. I saw "CustomStringConvertable" calls showing up on the profiler and thought that there was no reason in heaven that we should need to do that so much.